int32 ownerId

bool isReservationBroadcastOrDenial

# If reservationBroadcast
Rectangle[] reservations

